test/annotation_layer_builder_overrides.css

```diff
@@ -30,13 +30,21 @@
 }
 
 .annotationLayer :is(.linkAnnotation, .buttonWidgetAnnotation.pushButton) > a,
-.annotationLayer .popupTriggerArea {
+.annotationLayer .popupTriggerArea::after,
+.annotationLayer .fileAttachmentAnnotation .popupTriggerArea {
   opacity: 0.2;
   background: rgba(255, 255, 0, 1);
   box-shadow: 0 2px 10px rgba(255, 255, 0, 1);
 }
 
-.annotationLayer :is(.popupAnnotation, .popupWrapper) {
+.annotationLayer .popupTriggerArea::after {
+  display: block;
+  width: 100%;
+  height: 100%;
+  content: "";
+}
+
+.annotationLayer .popupAnnotation {
   display: block;
 }
```

Comment:
Is this block needed now, given the new `showPopup` test-only helper, since I *believe* that this CSS was used to show the Popup during ref-tests?